### PR TITLE
Document new denoisers and properties in Using Lightmap global illumination

### PR DESCRIPTION
**Pull request by Calinou**
This also updates List of features to be up-to-date with Godot 4.2's 3D rendering features.

Don't cherry-pick, as this documents features not present in 4.1.

**Project used for new images:** [test_lightmap_denoise.zip](https://github.com/godotengine/godot-docs/files/13035124/test_lightmap_denoise.zip)

- See https://github.com/godotengine/godot/pull/81659,
https://github.com/godotengine/godot/pull/82068 and
https://github.com/godotengine/godot/pull/82832.

### CODE REVIEW

**Review by clayjohn — APPROVED** [review]
Looks great!

File: about/list_of_features.rst [hunk 1]
```diff
@@ -203,7 +203,7 @@ Rendering
 - Directional lights (sun/moon). Up to 4 per scene.
 - Omnidirectional lights.
 - Spot lights with adjustable cone angle and attenuation.
-- Specular energy can be adjusted on a per-light basis.
+- Specular, indirect light and volumetric fog energy can be adjusted on a per-light basis.
```
Comment (AThousandShips) [inline about/list_of_features.rst:206]:
```suggestion
- Specular, indirect light, and volumetric fog energy can be adjusted on a per-light basis.
```

File: about/list_of_features.rst [hunk 2]
```diff
@@ -267,7 +273,8 @@ Rendering
 
 **Decals:**
 
-- Supports albedo, emissive, :abbr:`ORM (Occlusion Roughness Metallic)` and normal mapping.
+- :ref:`Supports albedo <doc_using_decals>`, emissive, :abbr:`ORM (Occlusion Roughness Metallic)`
```
Comment (AThousandShips) [inline about/list_of_features.rst:276]:
```suggestion
- :ref:`Supports albedo <doc_using_decals>`, emissive, :abbr:`ORM (Occlusion Roughness Metallic)`,
```

File: about/list_of_features.rst [hunk 3]
```diff
@@ -295,12 +302,16 @@ Rendering
...
+- Support for controlling how much fog rendering should affect the sky, with
+  separate controls for traditional and volumetric fog.
+- Support for making specific materials ignore fog.
 
 **Volumetric fog:**
 
 - Global :ref:`volumetric fog <doc_volumetric_fog>` that reacts to lights and shadows.
 - Volumetric fog can take indirect light into account when using VoxelGI or SDFGI.
 - Fog volume nodes that can be placed to add fog to specific areas (or remove fog from specific areas).
+  Supported shapes include box, ellipse, cone, cylinder and 3D texture-based density maps.
```
Comment (AThousandShips) [inline about/list_of_features.rst:314]:
```suggestion
  Supported shapes include box, ellipse, cone, cylinder, and 3D texture-based density maps.
```

**Review by mhilbrunner — APPROVED** [review]
LGTM besides the above

**Comment by mhilbrunner** [comment]
Thank you! Merged.

**Pull request merged**